.env.example

```diff
@@ -32,11 +32,11 @@ PRISMA_GENERATE_DATAPROXY=
 # ***********************************************************************************************************
 
 # - SHARED **************************************************************************************************
-NEXT_PUBLIC_WEBAPP_URL='http://localhost:3000'
+NEXT_PUBLIC_WEBAPP_URL='http://app.cal.local:3000'
```

Comment:
Had to use a different URL than "localhost" to play with cookie sessions between different subdomains. We should make everyone use this new scheme by default, it's needed to make orgs work locally.